tests/Backend/SOX/TriggersTest.php

```diff
@@ -137,4 +156,47 @@ public function testOtherRolesCannotUpdateOrdeleteTriggerInSOX(Client $client):
             $this->assertSame('You don\'t have access to the resource.', $e->getMessage());
         }
     }
+
+    public function testCreateTriggerInBranch(): void
+    {
+        $description = $this->generateDescriptionForTestObject();
+
+        $newBranch = $this->branches->createBranch($this->generateDescriptionForTestObject());
+        $developerClient = $this->getDeveloperStorageApiClient();
+        $developerBranchClient = $developerClient->getBranchAwareClient($newBranch['id']);
+        $branchBucketId = $this->initEmptyBucket(
+            $this->getTestBucketName($description),
+            self::STAGE_IN,
+            $description,
+            $developerBranchClient
+        );
+        $branchTableId = $developerBranchClient->createTableAsync(
+            $branchBucketId,
+            'languages',
+            new CsvFile(__DIR__ . '/../../_data/languages.csv')
+        );
+
+        $token = $this->getDefaultClient()->verifyToken();
+        try {
+            $this->getDefaultClient()->createTrigger([
```

Comment:
Test na to, že nejde volat v branchi máme?

Reply:
Testuje sa to v connection intergračním testem, ale klidně može byt, přidal sem.